modules/reissue-internal-certificates-central.adoc

```diff
@@ -7,21 +7,31 @@
 
 Central uses a built-in server certificate for authentication when communicating with other {product-title} services.
 This certificate is unique to your Central installation.
-The RHACS portal shows an information banner when the Central certificate is about to expire.
+The {product-title-short} portal shows an information banner when the Central certificate is about to expire.
 
 [NOTE]
 ====
-The information banner only appears 15 days before the certificate expiry date.
+The information banner only appears 15 days before the certificate expiration date.
 ====
 
+For {ocp} installations, beginning with {product-title-short} version 4.3.4, the Operator will automatically rotate all Central components' service TLS certificates 6 months before they expire. The following conditions apply:
+
+* The rotation of certificates in the secrets does not trigger the components to automatically reload them. However, reloads typically occur when the pod is replaced as part of an {product-title-short} upgrade or as a result of node reboots. If neither of those events happens at least every 6 months, you must restart the pods before the old (in-memory) service certificates expire. For example, you can delete the pods with an `app` label that contains one of the values of `central`, `central-db`, `scanner`, or `scanner-db`.
+
+* CA certificates are not updated. They are valid for 5 years.
+
+* The service certificates in the init bundles used by secured cluster components are not updated. You must rotate the init bundles at regular intervals.
+
+For Kubernetes installations, you must manually rotate TLS certificates. Instructions for manually rotating certificates are included in the following section.
+
 .Prerequisites
 
-* To reissue certificates, you must have `write` permission for the `ServiceIdentity` resource.
+* To reissue, or rotate, certificates, you must have `write` permission for the `ServiceIdentity` resource.
 
 .Procedure
 
-. Click on the link in the banner to download a YAML configuration file, which contains a new {ocp} secret, including the certificate and key values.
-. Apply the new YAML configuration file to the cluster where you have installed Central.
+. In the {product-title-short} portal, click on the link in the banner that announces the certificate expiration to download a YAML configuration file, which contains a new {ocp} secret. The secret includes the certificate and key values.
```

Comment:
@johannes94 - question about this section. This appears to be only for OCP. Is it the same process for Kubernetes platforms? (And am I correct in assuming the banner announcing the expiration is appearing in the RHACS GUI?) I have never seen the banner myself so I'm not sure.

Reply:
I think this also applies to Kubernetes platforms.

I've never seen the banner myself, but my understanding is also that it shows in RHACS GUI.